x-pack/platform/plugins/shared/fleet/public/applications/integrations/sections/epm/screens/home/card_utils.tsx

```diff
@@ -138,7 +140,7 @@ export const mapToCard = ({
     extraLabelsBadges,
   };
 
-  if (item.type === 'integration') {
+  if (item.type === 'integration' && !excludePolicyTemplatesFromInstallationStatus) {
```

Comment:
Could `item.installationInfo?.install_status` on line 144 reflect the installation status for individual policy template? so we don't have to check if it's a policy template or worry about if we need to exclude it or not.